.github/workflows/release.yml

```
@@ -45,19 +46,29 @@ jobs:
           r-version: ${{ matrix.r-version }}
       - uses: r-lib/actions/setup-pandoc@v2
       - uses: r-lib/actions/setup-tinytex@v2
-      - name: set up dependencies
-        shell: bash
-        env:
-          OS_NAME: ${{ matrix.os }}
-        run: $GITHUB_WORKSPACE/.github/setup.sh
+      - uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          extra-packages: any::rcmdcheck
+          needs: check
       - name: run R CMD CHECK tests
         shell: bash
         run: $GITHUB_WORKSPACE/test.sh
   all-successful:
     runs-on: ubuntu-latest
-    needs: [devel_test]
+    needs: [version_check, devel_test]
     steps:
       - name: Note that all tests succeeded
         run: echo "🎉"
+      - name: build tarball for submission to CRAN
+        run: R CMD Build .
```

```suggestion
        run: R CMD build .
```

I'm pretty sure this is lowercase.

---

for example: https://support.posit.co/hc/en-us/articles/200486518-Customizing-Package-Build-Options-in-the-RStudio-IDE